sycl/test-e2e/InvokeSimd/invoke_simd_smoke.cpp

```diff
@@ -3,8 +3,6 @@
 // ESIMD function.
 
 // TODO: enable on Windows once driver is ready
```

Comment:
Shouldn't this comment be moved to [sycl/test-e2e/InvokeSimd/lit.local.cfg](https://github.com/intel/llvm/pull/9417/files#diff-349f93e9d3914e67bd16eeff3e997c368cf78dded4e242d2827f4d41e7a5782c) ?